### PR TITLE
c# generating script code'

### PR DESCRIPTION
**Pull request by silverspoon19931120**
## Types of changes

_What types of changes does your code introduce to this project? Put an `x` in the boxes that apply._

- [ ] Bugfix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the pull request. If you are unsure about any of them, do not hesitate to ask. We are here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) documentation.
- [ ] I have rebased, and my remote branch is now up-to-date with the `master` branch of the upstream.
- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my changes are effective.
- [ ] I have added the necessary documentation (if appropriate).

### CODE REVIEW

**Comment by ledinhphuong** [comment]
Please fix lint issues fist, more detail at https://github.com/kobiton/appium-script-generator/actions/runs/7005335910/job/19076430635?pr=6

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "appium-script-schema"]
 	path = appium-script-schema
 	branch = master
-	url = git@github.com:kobiton/appium-script-schema.git
+	url = https://github.com/kobiton/appium-script-schema.git
```
Comment (ledinhphuong) [inline .gitmodules:4]: Could you please `rebase` your code base again I remembered that this change I added this [PR](https://github.com/kobiton/appium-script-generator/pull/4/files) last month.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 |
 <strong><a href="#debugging-code">Debugging code</a></strong>
 </p>
+
```
Comment (ledinhphuong) [inline CONTRIBUTING.md:9]: This is the same concern with my above comment in `.gitmodules` file.

File: __tests__/integration/generate-script.test.js [hunk 3]
```diff
@@ -11,6 +11,8 @@ const SERVICE_PROTO_FILE = path.resolve(
   __dirname, '../../src/appium-script-schema/generate-script.proto')
 const JAVA_JUNIT_INPUT_FILE = path.resolve(
   __dirname, '../resource/java-junit-input.json')
+  const CSHARP_NUNIT_INPUT_FILE = path.resolve(
+    __dirname, '../resource/csharp-nunit-input.json')
```
Comment (ledinhphuong) [inline __tests__/integration/generate-script.test.js:15]: Please correct the indentation.

File: __tests__/resource/csharp-nunit-input.json [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+  "sentAt": {
+    "low": 1105186912,
+    "high": 395,
+    "unsigned": true
+  },
+  "serverInfo": {
+    "apiUrl": "https://api.kobiton.com",
+    "portalUrl": "https://portal.kobiton.com",
+    "username": "kobitonadmin+appiumScriptGen",
+    "apiKey": "65530082-3eaa-44f8-adc5-e1d6d6a5a00c"
```
Comment (ledinhphuong) [inline __tests__/resource/csharp-nunit-input.json:11]:
```suggestion
    "apiKey": "your_kobiton_api_key"
```

File: package.json [hunk 5]
```diff
@@ -6,8 +6,8 @@
   "author": "Kobiton Inc.",
   "scripts": {
     "start": "bash ./scripts/register-grpc-server.sh; nodemon src/index.js --exec babel-node",
-    "test": "jest",
-    "lint": "eslint src",
+    "test": "jest --forceExit",
+    "lint": "eslint \"src/**/*.js\"",
```
Comment (ledinhphuong) [inline package.json:10]: The same concern as above

**Review by ledinhphuong — CHANGES_REQUESTED** [review]

**Comment by silverspoon19931120** [comment]
I fixed other issues,
But what do you mean this?
[image: image.png]

File: .gitmodules (hunk 1 above)
Comment (silverspoon19931120) [reply .gitmodules:4]: What do you mean?

**Pull request closed without merging**